src/form-elements/FormElementCompliance.tsx

```diff
@@ -243,6 +247,8 @@ function FormElementCompliance({
                 validationMessage={undefined}
                 value={typedValue?.notes}
                 element={notesElement}
+                isDirty={isTextDirty}
```

Comment:
I think this means that potentially you could have a situation where The invalid class is added to the whole element because the options are invalid, but the notes and files element do not yet show their appropriate validation... but I don't think its too big of a deal.

Reply:
Yeah, it's really there just to satisfy the props, I didn't want them optional/undefined

Reply:
Would it be better to just pass on the original, getter/setters instead of creating new dummy ones?

Reply:
Yeah, I can see that.  It's not actually possible to make them dirty until the radio is touched anyway, but I guess for future us this would be better.